src/herder/test/HerderTests.cpp

```diff
@@ -3258,7 +3253,7 @@ TEST_CASE("accept soroban txs after network upgrade", "[soroban][herder]")
 
     auto simulation =
         Topologies::core(4, 1, Simulation::OVER_LOOPBACK, networkID, [](int i) {
-            auto cfg = getTestConfig(i, Config::TESTDB_ON_DISK_SQLITE);
+            auto cfg = getTestConfig(i, Config::TESTDB_IN_MEMORY);
```

Comment:
The test arms a Soroban upgrade with a protocol 19 ledger, which makes the loadgen upgrade helper throw. Looking at this though, I think we can remove this test entirely. It's testing herder receiving soroban TXs post p19 upgrade. We don't have to test the upgrade since this is herder an we'll never see that behavior again. This is a super basic "do soroban txs make it into the block" test, and we have coverage elsewhere, so I'll just delete it.